utils/networking/src/lib.rs

```diff
@@ -16,3 +16,5 @@
 //! Various utils for working with async / tokio.
 
 pub mod broadcaster;
+pub mod ip_or_socket_address;
+pub use ip_or_socket_address::*;
```

Comment:
Perhaps only one of these two should be `pub`

Reply:
I've made the change in one of the follow-up PRs